trymito.io/pages/sitemap.xml.tsx

```diff
@@ -0,0 +1,114 @@
+        <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+            <!--We manually set the two URLs we know already-->
+            <url>
+                <loc>https://www.trymito.io/</loc>
+            </url>
+            <url>
+                <loc>https://www.trymito.io/plans</loc>
+            </url>
+            <url>
+                <loc>https://www.trymito.io/spreadsheet-automation</loc>
+            </url>
+            <url>
+                <loc>https://www.trymito.io/python-ai-tools</loc>
+            </url>
+            <url>
+                <loc>https://www.trymito.io/low-code-sql</loc>
+            </url>
+            <url>
+                <loc>https://www.trymito.io/data-app</loc>
+            </url>
+            <url>
+                <loc>https://www.trymito.io/infrastructure-integration-python-tool</loc>
+            </url>
+            <url>
+                <loc>https://www.trymito.io/industries/financial-services</loc>
+            </url>
+            <url>
+                <loc>https://www.trymito.io/industries/life-sciences</loc>
+            </url>
+            <url>
+                <loc>https://www.trymito.io/customers</loc>
+            </url>
+            <url>
+                <loc>https://www.trymito.io/security</loc>
+            </url>
+            <url>
+                <loc>https://www.trymito.io/blog</loc>
+            </url>
+            <url>
+                <loc>https://www.trymito.io/excel-to-python</loc>
+            </url>
+            <url>
+                <loc>https://www.trymito.io/teams</loc>
+            </url>
+            ${glossarySlugs.map(glossarySlug => {
+                return `
+                <url>
+                    <loc>${`${WEBSITE_HOST_URL}/excel-to-python/${glossarySlug}`}</loc>
+                </url>
+                `;
+            }).join('')}
+            ${blogPostSlugs.map(blogPostSlug => {
+                return `
+                <url>
+                    <loc>${`${WEBSITE_HOST_URL}/blog/${blogPostSlug}`}</loc>
+                </url>
+                `;
+            }).join('')}
+        </urlset>
+    `;
+}
+
+function SiteMap() {
+  // getServerSideProps will do the heavy lifting
+}
+
+export const getServerSideProps = async ({res}: any) => {
+    // Get the glossary pages
+    const pageContentsJsonArray = await getPageContentJsonArray()
+    const glossaryPageInfo = await getGlossaryPageInfo(pageContentsJsonArray)
+    const glossarySlugs = glossaryPageInfo.map((pageInfo) => pageInfo.slug.join('/'))
+
+    // Get the blog pages
+    const blogPostSlugs: string[] = []
+    const posts = await getPosts()
+    if (posts) {
+        // Redirect old slugs to new slugs
+        posts.forEach(post => {
+            if (SLUG_REDIRECTS[post.slug]) {
+                blogPostSlugs.push(SLUG_REDIRECTS[post.slug])
+            } else {
+                blogPostSlugs.push(post.slug)
+            }
+        })
+    }
```

Comment:
We should make this into a helper function so we don't have to remember to update it in both places if something changes.